### PR TITLE
Documentation revamp

### PR DESCRIPTION
**Pull request by juliettech13**
## Description

Added additional context to the documentation examples to provide further clarity and a smoother developer experience.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue) 
Not necessarily a bug, but more so an improvement to documentation.

## Checklist:

- [X] I have selected the correct base branch.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.
- [X] Any dependent changes have been merged and published in downstream modules.
- [X] I ran all tests with success and extended them if necessary.
- [N/A] I have updated the ``CHANGELOG.md`` file in the root folder of the package after the [UPCOMING] title and before the latest version.
- [N/A] I have tested my code on the test network.

### CODE REVIEW

File: modules/client/package.json [hunk 1]
```diff
@@ -61,6 +61,7 @@
   },
   "dependencies": {
     "@aragon/core-contracts-ethers": "^0.7.1-alpha",
+    "@aragon/sdk-client": "^0.20.0-beta",
```
Comment (Michael-A-Heuer) [inline modules/client/package.json:64]: Why was this added as a dependency?

**Review by heueristik — CHANGES_REQUESTED** [review]
Remove the package-lock.json files

File: modules/client/package.json (hunk 1 above)
Comment (heueristik) [inline modules/client/package.json]: Remove the self-dependency

**Review by heueristik — CHANGES_REQUESTED** [review]
- remove the two `package-lock.json` files ([package-lock.json](https://github.com/aragon/sdk/pull/152/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519) and [modules/client/package-lock.json](https://github.com/aragon/sdk/pull/152/files#diff-f9cd0e050c5442824303127287b1a03eb0b36a1124eddbb5107ee21c1cd8deb6)) that were added and 
- remove the `"@aragon/sdk-client": "^0.20.0-beta",` self-dependency from [modules/client/package.json](https://github.com/aragon/sdk/pull/152/files#diff-952369e42cca7aee14a7fd3ec1138bb8689b5a8ace997b4e947c3160a1bc948a)

File: modules/client/examples/00-setup/00-getting-started.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+/* MARKDOWN
+Update the context with new parameters if you wish to throughout your app.
+
+```tsx
+context.set({ network: 1 });
+context.set({ signer: new Wallet("private key") }); // if you're using wagmi library, you can also get the signer through their [`useSigner` method](https://wagmi.sh/react/hooks/useSigner) inside a `useEffect` hook.
+context.setFull(contextParams);
+```
+*/
```
Comment (brickpop) [inline modules/client/examples/00-setup/00-getting-started.tsx]: This could be outside of the MARKDOWN + TSX section

File: modules/client/examples/00-setup/00-getting-started.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
+/* MARKDOWN
```
Comment (brickpop) [inline modules/client/examples/00-setup/00-getting-started.tsx:1]: Looks great :)

But don't you think that's easier to have the examples section as a more of a self-contained code snippets approach?

This new format feels more like a tutorial than a simple examples file. Happy to keep it like this

File: modules/client/examples/00-setup/01-context-plugin.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+/* MARKDOWN
+Update the context with new parameters if you wish to throughout your app.
+
+```tsx
+contextPlugin.set({ network: 1 });
+contextPlugin.set({ signer: new Wallet("other private key") });
+contextPlugin.setFull(contextParams);
+
+console.log({ contextPlugin });
+```
```
Comment (brickpop) [inline modules/client/examples/00-setup/01-context-plugin.tsx]: This could be simply moved out below as well

File: modules/client/examples/01-client/09-pin-metadata.ts [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+const metadataUri = await client.methods.pinMetadata(metadata);
+console.log({ metadataUri });
+
+/* MARKDOWN
+Returns:
+
+```javascript
+  "ipfs://Qm..."
+```
+*/
```
Comment (brickpop) [inline modules/client/examples/01-client/09-pin-metadata.ts]: Needed? This should be a JS object, if so

File: modules/client/examples/02-tokenVoting-client/05-can-vote.ts [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+// Returns true or false depending on whether the address can vote in the specific proposal.
+const canVote: boolean = await tokenVotingClient.methods.canVote(canVoteParams);
+console.log({ canVote });
+
+/* MARKDOWN
+Returns:
+
+```javascript
+  true
+```
```
Comment (brickpop) [inline modules/client/examples/02-tokenVoting-client/05-can-vote.ts:36]: Same as above

File: modules/client/examples/02-tokenVoting-client/09-get-voting-settings.ts [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+const pluginSettings: VotingSettings | null = await tokenVotingClient.methods.getVotingSettings(pluginAddress);
+console.log({ pluginSettings });
+
+/* MARKDOWN
+Returns:
+
+```json
+  {
+    minDuration: 10000, // 10 seconds
```
Comment (brickpop) [inline modules/client/examples/02-tokenVoting-client/09-get-voting-settings.ts]: @josemarinas is this the case?   I was thinking that the input parameter was in seconds, not miliseconds. This should 100% match

File: modules/client/examples/03-multisig-client/05-can-approve.ts [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+// Instantiate a Multisig client.
+const client = new MultisigClient(contextPlugin);
+
+const canApproveParams: CanApproveParams = {
+  pluginAddress: "0x1234567890123456789012345678901234567890",
+  addressOrEns: "0x1234567890123456789012345678901234567890",
+  proposalId: BigInt(0)
+};
+
+// Checks whether the signer of the transaction can approve a given proposal.
```
Comment (brickpop) [inline modules/client/examples/03-multisig-client/05-can-approve.ts]: Judging by the parameters described above, I think this is out of sync. The given `addressOfEns` should be the one that can approve or not?

@josemarinas ?

File: modules/client/examples/03-multisig-client/06-can-execute.ts [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Instantiate a Multisig client.
+const multisigClient: MultisigClient = new MultisigClient(contextPlugin);
+
+const canExecuteParams: CanExecuteParams = {
+  pluginAddress: "0x1234567890123456789012345678901234567890", // the address of the plugin contract itself.
+  proposalId: BigInt(0)
+};
+
+// Checks whether the signer of the transaction can execute a given proposal.
```
Comment (brickpop) [inline modules/client/examples/03-multisig-client/06-can-execute.ts:20]: This feels inaccurate. Approved proposals can be approved by anyone, being a signer or not

Comment (brickpop) [reply modules/client/examples/03-multisig-client/06-can-execute.ts:20]: Is this taken from the source code? If so, we also need to fix it there

File: modules/client/examples/04-addresslistVoting-client/00-installation.ts [hunk 10]
```diff
@@ -76,6 +80,6 @@ for await (const step of steps) {
         break;
```
Comment (brickpop) [inline modules/client/examples/04-addresslistVoting-client/00-installation.ts:81]: Now, the DONE step also reports the addresses of the plugins installed

File: modules/client/examples/04-addresslistVoting-client/06-can-vote.ts [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+} from "@aragon/sdk-client";
+import { context } from "../00-setup/00-getting-started";
+
+// Instantiate a plugin context from the aragonOSx SDK context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create an AddresslistVoting client.
+const addresslistVotingClient = new AddresslistVotingClient(contextPlugin);
+
+const voteParams: ICanVoteParams = {
+  address: "0x1234567890123456789012345678901234567890",
```
Comment (brickpop) [inline modules/client/examples/04-addresslistVoting-client/06-can-vote.ts]: @josemarinas this is inconsistent with the `addressOrEns` above. 

To avoid confusions, I would name all of them like `targetAddress`. If there's an ENS, we silently resolve it

File: modules/client/examples/05-encoders/03-freeze-permissions.ts [hunk 12]
```diff
@@ -0,0 +1,38 @@
+/* MARKDOWN
```
Comment (brickpop) [inline modules/client/examples/05-encoders/03-freeze-permissions.ts]: This permission no longer exists. The encoder should not exist either.

@josemarinas didn't we remove these encoder/decoder?

File: modules/client/examples/05-encoders/04-withdraw.ts [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+import { context } from "../00-setup/00-getting-started";
+
+// Instantiates an aragonOSx SDK client.
+const client: Client = new Client(context);
+
+const withdrawParams: IWithdrawParams = {
+  recipientAddress: "0x1234567890123456789012345678901234567890",
+  amount: BigInt(10),
+  tokenAddress: "0x1234567890123456789012345678901234567890", // contract address of the token to be withdrawn. if left empty, ETH will be withdrawn.
+  reference: "reason for the withdrawal"
```
Comment (brickpop) [inline modules/client/examples/05-encoders/04-withdraw.ts]: This has changed, after the withdraw function no longer exists. Also, the parameters now specify what token type and there's no `reference` 🙏

File: modules/client/examples/05-encoders/06-update-plugin-settings-address-list.ts [hunk 14]
```diff
@@ -0,0 +1,32 @@
+/* MARKDOWN
```
Comment (brickpop) [inline modules/client/examples/05-encoders/06-update-plugin-settings-address-list.ts]: Wouldn't you make the difference between general DAO actions and plugin-specific actions?

File: modules/client/examples/06-decoders/03-freeze-permission.ts [hunk 15]
```diff
@@ -0,0 +1,28 @@
+/* MARKDOWN
```
Comment (brickpop) [inline modules/client/examples/06-decoders/03-freeze-permission.ts]: Same as aabove, this should be removed completely

File: modules/client/examples/06-decoders/04-withdraw.ts [hunk 16]
```diff
@@ -0,0 +1,30 @@
...
+
+/* MARKDOWN
+Returns:
+
+```json
+{
+  recipientAddress: "0x1234567890123456789012345678901234567890",
+  amount: 10n,
+  tokenAddress: "0x1234567890123456789012345678901234567890",
+  reference: "test"
```
Comment (brickpop) [inline modules/client/examples/06-decoders/04-withdraw.ts]: As above, reference is gone. Returning a token type now. Token Address being optional

File: modules/client/package.json (hunk 1 above)
Comment (brickpop) [reply modules/client/package.json]: +1

**Review by brickpop — COMMENTED** [review]
Thank you for the huge work! 

In addition to the many comments, please let's remove package-lock.json since we always use `yarn`

File: modules/client/package.json (hunk 1 above)
Comment (juliettech13) [reply modules/client/package.json]: Bizzaaaaarre. Great shout!

File: modules/client/examples/00-setup/00-getting-started.tsx (hunk 3 above)
Comment (juliettech13) [reply modules/client/examples/00-setup/00-getting-started.tsx:1]: Yesss. I debated with this myself. Im afraid, as it is right now, this is the only documentation devs will have coming in to the SDK. So I felt like it needed a bit more context before leaving them in the wild. 
Once we get more tutorials going on this, we can certainly scope back to merely self-contained code snippets.

File: modules/client/examples/00-setup/00-getting-started.tsx (hunk 2 above)
Comment (juliettech13) [reply modules/client/examples/00-setup/00-getting-started.tsx:74]: How come? It's inside Markdown block?

Comment (juliettech13) [reply modules/client/examples/00-setup/00-getting-started.tsx]: OH duh. Smart man.
smart

File: modules/client/examples/02-tokenVoting-client/09-get-voting-settings.ts (hunk 7 above)
Comment (juliettech13) [reply modules/client/examples/02-tokenVoting-client/09-get-voting-settings.ts]: @josemarinas 👀

File: modules/client/examples/03-multisig-client/05-can-approve.ts (hunk 8 above)
Comment (juliettech13) [reply modules/client/examples/03-multisig-client/05-can-approve.ts]: @josemarinas 👀

File: modules/client/examples/01-client/09-update-allowance.ts [hunk 17]
```diff
@@ -0,0 +1,44 @@
+/* MARKDOWN
```
Comment (josemarinas) [inline modules/client/examples/01-client/09-update-allowance.ts]: This is duplicated, we have ensure-allowance and update-allowance

rename this to `08-update-allowance` and remove the oter one since the function was renamed from `ensuraAllowance` to `updateAllowance`

File: modules/client/examples/00-setup/00-getting-started.tsx [hunk 18]
```diff
@@ -0,0 +1,74 @@
...
+# Getting Started with the aragonOSx SDK
```
Comment (josemarinas) [inline modules/client/examples/00-setup/00-getting-started.tsx]: Please run a `yarn examples` to generate the `examples.md` file

File: modules/client/examples/05-encoders/04-withdraw.ts (hunk 13 above)
Comment (juliettech13) [reply modules/client/examples/05-encoders/04-withdraw.ts]: In this scenario though, I see that the `WithdrawParams` interface also doesn't have the `recepientAddress` attribute. How is it supposed to know where to withdraw the funds to? 
@brickpop @josemarinas

File: modules/client/examples/05-encoders/06-update-plugin-settings-address-list.ts (hunk 14 above)
Comment (juliettech13) [reply modules/client/examples/05-encoders/06-update-plugin-settings-address-list.ts]: LOVE. Will implement!

File: modules/client/examples/01-client/01-create-dao.ts [hunk 19]
```diff
@@ -0,0 +1,58 @@
...
+  }]
+};
+
+// Through pinning the metadata in IPFS, we can get the IPFS URI. You can read more about it here: https://docs.ipfs.tech/how-to/pin-files/
+const metadataUri = await client.methods.pinMetadata(metadata);
+
+const createDaoParams: CreateDaoParams = {
+  metadataUri,
+  ensSubdomain: "my-org", // my-org.dao.eth
+  plugins: [] // may be left as an empty array, but we recommend adding at least 1 governance plugin upon creation.
```
Comment (josemarinas) [inline modules/client/examples/01-client/01-create-dao.ts]: This cannot be empty, you cannot create a dao without at least one plugin, the transaction will fail

File: modules/client/examples/01-client/01-create-dao.ts [hunk 20]
```diff
@@ -0,0 +1,58 @@
...
+};
+
+// Estimate how much gas the transaction will cost.
+const estimatedGas: GasFeeEstimation = await client.estimation.createDao(createDaoParams);
```
Comment (josemarinas) [inline modules/client/examples/01-client/01-create-dao.ts:69]: I'm not sure but maybe we should make clear that this is step is totally optional

File: modules/client/examples/01-client/01-create-dao.ts [hunk 21]
```diff
@@ -0,0 +1,58 @@
...
+    switch (step.key) {
+      case DaoCreationSteps.CREATING:
+        console.log(step.txHash);
+        break;
+      case DaoCreationSteps.DONE:
+        console.log(step.address);
+        break;
+    }
+  } catch (err) {
+    console.error({ err });
```
Comment (josemarinas) [inline modules/client/examples/01-client/01-create-dao.ts]: Just out of curiosity, why logging `{ err: err }` instead of just `err`?

File: modules/client/examples/01-client/02-deposit-eth.ts [hunk 22]
```diff
@@ -1,34 +1,32 @@
 /* MARKDOWN
-### Depositing ETH to a DAO
-
-Handles the flow of depositing the native token to an Aragon DAO.
+### Deposit ETH to a DAO
 
+Handles the flow of depositing the native EVM token (when in mainnet, it's ETH) to an Aragon OSx DAO.
```
Comment (josemarinas) [inline modules/client/examples/01-client/02-deposit-eth.ts:4]: The token deposited does not depend on the network you are, it depends on the token type that you select on the params, if its native i'll be ETH, if its erc20 it will be any ERC20 token and you need to specify the address of the token contract

Comment (josemarinas) [reply modules/client/examples/01-client/02-deposit-eth.ts:4]: nevermind just saw that they are 2 diferent files for each type of deposit, resolving

File: modules/client/examples/01-client/05-get-dao.ts [hunk 23]
```diff
@@ -1,19 +1,26 @@
 /* MARKDOWN
-### Loading DAO details
-
-Handles retrieving DAO metadata using its address or ENS domain.
+### Get DAO details
 
+Gets a specific DAO's metadata using its address or ENS domain.
```
Comment (josemarinas) [inline modules/client/examples/01-client/05-get-dao.ts]: This comes from the original documentation, but maybe since we call metadata to a field of the dao object "Gets a specific DAO's metadata" can be confusing

File: modules/client/examples/01-client/06-get-transfers.ts [hunk 24]
```diff
@@ -1,36 +1,42 @@
 /* MARKDOWN
-### Loading DAO activity
+### Get transfers from DAO's activity
 
-Retrieves the list of asset transfers to and from the given DAO (by default,
-from ETH, DAI, USDC and USDT, on Mainnet)
+Gets the list of asset transfers to and from DAOs.
+If passed a `daoAddressOrEns`, will only retrieve transfers for that DAO. Otherwise, it returns for all DAOs.
+
+By default, retrieves ETH, DAI, USDC and USDT, on Mainnet).
```
Comment (josemarinas) [inline modules/client/examples/01-client/06-get-transfers.ts:7]: Not sure if this is true, @brickpop we retrieve all transfers that are made with the deposit or withdraw functions, not only ETH, DAI, USDC and UST no?

File: modules/client/examples/01-client/08-ensure-allowance.ts [hunk 25]
```diff
@@ -0,0 +1,42 @@
...
+const client: Client = new Client(context);
+
+const ensureAllowanceParams = {
+  amount: BigInt(10), // amount in wei
+  daoAddress: "0x1234567890123456789012345678901234567890",
+  tokenAddress: "0x1234567890123456789012345678901234567890" // token contract adddress
+};
+
+// Ensure the token's approval allowance is enough.
+const steps = client.methods.ensureAllowance(ensureAllowanceParams);
```
Comment (josemarinas) [inline modules/client/examples/01-client/08-ensure-allowance.ts]: this was renamed to `updateAllowance`

File: modules/client/examples/02-tokenVoting-client/01-installation.ts [hunk 26]
```diff
@@ -0,0 +1,113 @@
...
+  },
+  useToken: {
+    address: "0x23847102387419273491234", // contract address of the token to use as the voting token
+  },
+};
+
+// These would be the plugin params if you need to mint a new token for the DAO to enable TokenVoting.
+const pluginInitParams2: ITokenVotingPluginInstall = {
+  votingSettings: {
+    minDuration: 60 * 60 * 24 * 2, // seconds
```
Comment (josemarinas) [inline modules/client/examples/02-tokenVoting-client/01-installation.ts]: The minimum duration is 3600, make sure we leave this clear

File: modules/client/examples/02-tokenVoting-client/01-installation.ts [hunk 27]
```diff
@@ -0,0 +1,113 @@
...
+  metadataUri,
+  ensSubdomain: "my-org", // my-org.dao.eth
+  plugins: [tokenVotingInstallPluginItem1, tokenVotingInstallPluginItem2] // optional, this will determine the plugins installed in your DAO upon creation.
+};
+
+// Estimate how much gas the transaction will cost.
+const estimatedGas: GasFeeEstimation = await client.estimation.createDao(createParams);
+console.log({ avg: estimatedGas.average, max: estimatedGas.max });
+
+// Create the DAO with the two token voting plugins installed. This means that the DAO will be able to use either of the two tokens to vote.
```
Comment (josemarinas) [inline modules/client/examples/02-tokenVoting-client/01-installation.ts]: Maybe its me but this can be misunderstood, if you create a dao with 2 voting plugins you can vote with the token A in the plugin A and with the token B in the plugin B but you cannot vote with the Token A in the plugin B. If you feel that the prhase is crear enough feel free to resolve

File: modules/client/examples/02-tokenVoting-client/01-installation.ts [hunk 28]
```diff
@@ -0,0 +1,113 @@
...
+// You can also create a DAO with only one of the plugins installed or none.
```
Comment (josemarinas) [inline modules/client/examples/02-tokenVoting-client/01-installation.ts]: AFAIK creating a dao without plugins is not an option @brickpop or @mathewmeconry can you confirm?

File: modules/client/examples/02-tokenVoting-client/02-create-proposal.ts [hunk 29]
```diff
@@ -0,0 +1,87 @@
...
+  minParticipation: 0.25, // 25%
+  supportThreshold: 0.5, // 50%
+  minProposerVotingPower: BigInt("5000"), // default 0
+  votingMode: VotingMode.EARLY_EXECUTION, // default STANDARD, other options: EARLY_EXECUTION, VOTE_REPLACEMENT
+};
+
+const proposalParams: CreateMajorityVotingProposalParams = {
+  pluginAddress,
+  metadataUri,
+  actions: [configActionPrarms], // this is optional. if you don't want to pass any actions, simply pass an empty array `[]`.
```
Comment (josemarinas) [inline modules/client/examples/02-tokenVoting-client/02-create-proposal.ts]: This is not whoy yo pass an action to a proposal, you first need to encode it:
```ts
const action = client.encoding.updateVotingSettingsAction(configActionParams) // I think this is te correct action for updating the Voting settings, but check just in case
```

File: modules/client/examples/02-tokenVoting-client/03-create-proposal-with-actions.ts [hunk 30]
```diff
@@ -0,0 +1,85 @@
...
+  minProposerVotingPower: BigInt("5000"), // default 0
+  votingMode: VotingMode.EARLY_EXECUTION // default standard, other options: EARLY_EXECUTION, VOTE_REPLACEMENT
+};
+
+// The contract address of the token voting plugin you have installed in your DAO
+const pluginAddress: string = "0x1234567890123456789012345678901234567890";
+
+// Update the configuration of the plugin.
+const configAction = tokenVotingClient.encoding.updatePluginSettingsAction(
+  configActionParams,
```
Comment (josemarinas) [inline modules/client/examples/02-tokenVoting-client/03-create-proposal-with-actions.ts]: First, goes the `pluginAddress`, then the params
```ts
  const configAction = tokenVotingClient.encoding.updatePluginSettingsAction(
  pluginAddress,
  configActionParams,
);
```

File: modules/client/examples/02-tokenVoting-client/04-vote-on-proposal.ts [hunk 31]
```diff
@@ -0,0 +1,44 @@
...
+import { context } from "../00-setup/00-getting-started";
+
+// Instantiate a plugin context from the Aragon OSx SDK context.
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+
+// Create a TokenVoting client.
+const tokenVotingClient: TokenVotingClient = new TokenVotingClient(contextPlugin);
+
+const voteParams: IVoteProposalParams = {
+  pluginAddress: "0x1234567890123456789012345678901234567890", // contract address of the TokenVoting plugin installed into the DAO
```
Comment (josemarinas) [inline modules/client/examples/02-tokenVoting-client/04-vote-on-proposal.ts]: This parameter was deleted, now we just need the proposalId.

```ts
const voteParams: IVoteProposalParams = {
  proposalId: "0x1234567890123456789012345678901234567890_0x0",
  vote: VoteValues.YES // other options: NO, ABSTAIN
};
```

File: modules/client/examples/01-client/05-get-dao.ts (hunk 23 above)
Comment (juliettech13) [reply modules/client/examples/01-client/05-get-dao.ts]: Great catch! Fixed locally.

File: modules/client/examples/01-client/08-ensure-allowance.ts (hunk 25 above)
Comment (juliettech13) [reply modules/client/examples/01-client/08-ensure-allowance.ts]: Alright - so I'll delete this example and only keep `updateAllowance` example, right? @josemarinas

File: modules/client/examples/02-tokenVoting-client/06-get-members.ts [hunk 32]
```diff
@@ -0,0 +1,32 @@
...
+import { context } from "../00-setup/00-getting-started";
+
+// Instantiate a plugin context from the Aragon OSx SDK context.
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create a TokenVoting client
+const tokenVotingClient: TokenVotingClient = new TokenVotingClient(contextPlugin);
+
+const daoAddressorEns: string = "0x12345384572394756239846529574932532985"; // or my-dao.dao.eth
+
+const members: string[] = await tokenVotingClient.methods.getMembers(daoAddressorEns);
```
Comment (josemarinas) [inline modules/client/examples/02-tokenVoting-client/06-get-members.ts]: Here you dont pass the dao address you pass the pluginAddress

File: modules/client/examples/02-tokenVoting-client/07-get-proposal.ts [hunk 33]
```diff
@@ -1,30 +1,32 @@
...
-// Create a plugin context from the simple context
+// Instantiate a plugin context from an Aragon OSx SDK context.
 const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
-// Create a TokenVoting client
-const client = new TokenVotingClient(contextPlugin);
+// Create a TokenVoting client.
+const tokenVotingClient: TokenVotingClient = new TokenVotingClient(contextPlugin);
+
+// The address of the proposal you want to retrieve.
+const proposalId: string = "0x12345238513498562394651375914323423";
```
Comment (josemarinas) [inline modules/client/examples/02-tokenVoting-client/07-get-proposal.ts]: El formato correcto de proposal id seria algo del estilo `0x12345238513498562394651375914323423_0x1`

File: modules/client/examples/02-tokenVoting-client/09-get-voting-settings.ts (hunk 7 above)
Comment (josemarinas) [reply modules/client/examples/02-tokenVoting-client/09-get-voting-settings.ts]: AFAIK, it's seconds

File: modules/client/examples/02-tokenVoting-client/10-get-token.ts [hunk 34]
```diff
@@ -0,0 +1,48 @@
...
+```json
+  {
+    address: "0x123456789000987654323112345678900987654321",
+    name: "Token",
+    symbol: "TOK",
+    decimals: 18
+  }
+```
+Or:
+```json
```
Comment (josemarinas) [inline modules/client/examples/02-tokenVoting-client/10-get-token.ts:41]: ERC721 coming soon

File: modules/client/examples/02-tokenVoting-client/07-get-proposal.ts (hunk 33 above)
Comment (juliettech13) [reply modules/client/examples/02-tokenVoting-client/07-get-proposal.ts]: Porque? jaj que es el `_0x1`?

Comment (juliettech13) [reply modules/client/examples/02-tokenVoting-client/07-get-proposal.ts]: Is this for every time a `proposalId` is called?

File: modules/client/examples/01-client/06-get-transfers.ts (hunk 24 above)
Comment (brickpop) [reply modules/client/examples/01-client/06-get-transfers.ts:7]: This depends on subgraph, all ERC20 outgoing transfers should be monitored

File: modules/client/examples/02-tokenVoting-client/07-get-proposal.ts (hunk 33 above)
Comment (brickpop) [reply modules/client/examples/02-tokenVoting-client/07-get-proposal.ts]: The proposal ID includes the plugin address and the proposal counter, so that the plugin knows which one it is. There is no `the address of the proposal`, it's simply the proposal ID

File: modules/client/examples/03-multisig-client/06-can-execute.ts (hunk 9 above)
Comment (juliettech13) [reply modules/client/examples/03-multisig-client/06-can-execute.ts:20]: Fixed!

File: modules/client/examples/02-tokenVoting-client/07-get-proposal.ts (hunk 33 above)
Comment (juliettech13) [reply modules/client/examples/02-tokenVoting-client/07-get-proposal.ts]: Will fix for all then! Thanks for the insight @brickpop

File: modules/client/examples/02-tokenVoting-client/01-installation.ts (hunk 28 above)
Comment (juliettech13) [reply modules/client/examples/02-tokenVoting-client/01-installation.ts]: I've ensured all DAO creation processes have a plugin. Either way, let us know for guidance. The SDK didn't throw TS errors on an empty array.

**Review by brickpop — APPROVED** [review]

**Review by brickpop — APPROVED** [review]
